tests/examples/awful/wallpaper/tiled2.lua

```diff
@@ -0,0 +1,30 @@
+--DOC_NO_USAGE --DOC_GEN_IMAGE
```

Comment:
Maybe it's too late right now for my brain to understand this example... What did you do to make the centered star be white while others are black?

Reply:
whoops, that was a bug in the tile container I forgot to fix. I added a commit for it

Reply:
I feel a little sad it's gone... It was so cool!